.gitignore

```diff
@@ -12,3 +12,4 @@ docs/**/*.html
 /script/yaml_generator/env/
 __pycache__/
 .tidyall.d/
+.vscode/
```

Comment:
Leftover?

Reply:
No, intentional! I have so much issue with `.vscode`: also having configured it in the home, git status does not show it,but in commit is included and I have to reset all. With this addition instead all ok.

Reply:
Please not. If you want to push this change, we need to include a far larger audience, as this affects everyone working in this repository.

If you want to do that, please say so, but then we need to involve the other squads and users of this repository in the review as well.

Reply:
OK, I will keep my problems for me only and will revert that file.

Reply:
agree with Felix and Pavel

Reply:
Original restored